runtime/src/accounts_db.rs

```diff
@@ -603,15 +603,27 @@ impl AccountsDB {
 
         // Recalculate reclaims with new purge set
         let mut reclaims = Vec::new();
+        let mut dead_keys = Vec::new();
         for pubkey in purges.keys() {
-            reclaims.extend(accounts_index.purge(&pubkey));
+            let purge = accounts_index.purge(&pubkey);
```

Comment:
I think `.1` and `.0` isn't readable... So how about this using destructring?

```rust
let mut all_reclaims = Vec::new();
for ...
    let (reclaims, is_empty) = accounts_index...
    all_reclaims.extend(reclaims);
    if (is_empty) {
        dead_keys.push(...
    }
```

Reply:
Yea. That's better. Thanks.